src/components/molecules/NavBar/NavBar.scss

```diff
@@ -286,7 +286,7 @@ $border-radius: 28px;
 
 .schedule-dropdown-body {
   position: fixed;
-  z-index: z(schedule);
+  z-index: z(navbar-schedule);
```

Comment:
Nice catch!

I expected our `z()` helper to error-out when there isn't a matching key, but seems we only emit a warning, then default it:

- https://github.com/sparkletown/sparkle/blob/staging/src/scss/constants.scss#L109-L116

Could I please get you to also make a change to the `z()` helper to swap `@warn`/the default value for `@error`:

- https://sass-lang.com/documentation/at-rules/error

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1202/commits/5ae3d61bad5f17cd0190f10b24913231d6ad7bf1